From a4940255d74bce3a7e9519398e4e343eba9c0227 Mon Sep 17 00:00:00 2001
From: Michal Simek <michal.simek@xilinx.com>
Date: Thu, 1 May 2014 09:36:38 -0700
Subject: [PATCH 033/509] ARM: zynq: Rename zynq_platform_cpu_die to
 zynq_cpu_die

https://github.com/analogdevicesinc/linux.git xcomm_zynq
commit 0f75724bbf37de30bd71949a1b242c9e009b021b

Rename zynq_platform_cpu_die() to zynq_cpu_die()
to match the name of the other Zynq SMP ops.
Also use kernel-doc format to document this function.

Signed-off-by: Michal Simek <michal.simek@xilinx.com>
Signed-off-by: Zhong Hongbo <hongbo.zhong@windriver.com>
---
 arch/arm/mach-zynq/platsmp.c |   11 ++++++-----
 1 files changed, 6 insertions(+), 5 deletions(-)

diff --git a/arch/arm/mach-zynq/platsmp.c b/arch/arm/mach-zynq/platsmp.c
index 849a0de..87ea838 100644
--- a/arch/arm/mach-zynq/platsmp.c
+++ b/arch/arm/mach-zynq/platsmp.c
@@ -124,12 +124,13 @@ static int zynq_cpu_kill(unsigned cpu)
 	return 1;
 }
 
-/*
- * platform-specific code to shutdown a CPU
+/**
+ * zynq_cpu_die - platform-specific code to shutdown a CPU
+ * @cpu: cpu number
  *
- * Called with IRQs disabled
+ * Note: Called with IRQs disabled on the dying CPU.
  */
-static void zynq_platform_cpu_die(unsigned int cpu)
+static void zynq_cpu_die(unsigned int cpu)
 {
 	/*
 	 * there is no power-control hardware on this platform, so all
@@ -146,7 +147,7 @@ struct smp_operations zynq_smp_ops __initdata = {
 	.smp_prepare_cpus	= zynq_smp_prepare_cpus,
 	.smp_boot_secondary	= zynq_boot_secondary,
 #ifdef CONFIG_HOTPLUG_CPU
-	.cpu_die		= zynq_platform_cpu_die,
+	.cpu_die		= zynq_cpu_die,
 	.cpu_kill		= zynq_cpu_kill,
 #endif
 };
-- 
1.7.5.4

